superchain/superchain.go

```diff
@@ -56,18 +79,18 @@ const (
 	Frontier SuperchainLevel = 1
 )
 
-type ChainConfig struct {
+type RollupConfig struct {
```

Comment:
Why was it necessary to rename this?

Reply:
Not necessary, just seemed like a more descriptive name to distinguish between a "SuperchainConfig" and a "RollupConfig". "ChainConfig" felt ambiguous to me but can revert if you want

Reply:
I think "rollup config" is already used for `rollup.json`. I don't feel hugely strongly but I think the best thing to do is pinch this renaming change off into a different PR. If you revert it, it should reduce the diff here, which I think is helpful. Optional then whether you want to make another PR with the change and we can get some more opinions.

Reply:
Reverted